indico/modules/events/registration/util.py

```diff
@@ -309,7 +309,6 @@ def generate_csv_from_registrations(registrations, regform_items, special_items)
                 departure_date = data[item.id].friendly_data.get('departure_date') if item.id in data else None
                 registration_dict[key] = _prepare_data(format_date(departure_date) if departure_date else '')
             else:
-                key = '{}_{}'.format(item.title.encode('utf-8'), item.id)
                 registration_dict[key] = _prepare_data(data[item.id].friendly_data if item.id in data else '')
         if 'reg_date' in special_items:
             registration_dict['Registration date'] = format_datetime(registration.submitted_dt)
```

Comment:
for name, (title, fn) in special_item_mapping.iteritems():
    if name not in special_item:
        continue
    value = fn(registration)
    if isinstance(value, unicode)
        value = value.encode('utf-8')
    registration_dict[title] = value